### DISCUSSION TITLE
Upgrade to 1.14.1 broke link between customer/project/activity

### DISCUSSION THREAD

**Discussion by eransurg** [kimai/kimai#2599] [On-Premise: Installation & Updates] [answered]
After upgrading to version 1.14.1, when I select a customer from the drop-down list I get all the projects and not the project related to the specific customer I selected. Also, the activity drop-down is empty, even if I select the correct project from the entire list.
No errors detected during the upgrade.

**Comment by kevinpapst** [comment, marked as answer]
Check your javascript console, you will see some kind of error. This error prevents reloading the list of projects.
Docker? Forgot to update public/ ?
Git? Check if `git status` reports partial update

**Reply by eransurg** [reply]
Hi,
Just figured out the php 7.2 is not supported anymore.
Upgraded to 7.4 and everything went back to normal.
Thanks!

**Reply by kevinpapst** [reply]
Aha. Hm, strange that `composer install` did not show a warning, but glad that it was so quick to fix
